test/index-spec.js

```diff
@@ -0,0 +1,55 @@
+import * as mocha from 'mocha';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import io from 'socket.io-client';
+
+import Server from '../src/server';
+
+const { assert, expect } = chai;
+const options = null;
+const user1 = { name: 'user1' };
+
+const server = new Server();
+server.start()
+  .then(() => {
+    chai.use(chaiHttp);
+
+    mocha.describe('GET /echo test', () => {
+      mocha.it('should echo hello', (done) => {
+        const echoUrl = '/echo?msg=hello';
```

Comment:
If http server is not necessary, this codes are deleted :(

Reply:
'echo' and test of 'echo' are just for verifying that server start
In my opinion, Signaling server doesn't support http request.
Therefore, I agree that http server is not necessary.
So, It is more acceptable to delete 'echo' and test of 'echo' in the near future